protected/humhub/modules/like/assets/LikeAsset.php

```diff
@@ -23,6 +23,13 @@ class LikeAsset extends AssetBundle
      */
     public $sourcePath = '@like/resources';
 
+    /**
+     * @inheritdoc
+     */
+    public $css = [
+        'css/humhub.like.css'
```

Comment:
Good, I will move this CSS code to `static/less/theme` and I will these extra lines.

Thank you for your help!

Reply:
[All other can* methods are part of the content class](https://github.com/humhub/humhub/pull/4339#discussion_r483680644)
Could you please specify the path in which file I should move this method, please?
I did it by `protected/humhub/modules/comment/Module.php` example

[Example](https://github.com/humhub/humhub/blob/master/protected/humhub/modules/comment/Module.php#L69-#L91)


> Doesn't it make more sense to use a global user group permission instead of a container group permission?
Something like this:
`Yii::$app->user->can(ManageSpaces::class)`? or how?
Thank you! Hope you may understand me.